app/views/index.scala.html

```diff
@@ -196,49 +196,69 @@
                         </a>
                     </div>
-                }
-            </section>
-            <section class="collaborator-images-container">
+                    <div class="collaborator-image">
+                        <a id="uic-img-link" href="https://idhd.ahs.uic.edu/">
+                            <img src='@routes.Assets.at("assets/uic_logo.png")' alt='@Messages("logo.uic")'>
+                        </a>
+                    </div>
+                    <div class="collaborator-image">
+                        <a id="ligapeatonal-img-link" href="https://ligapeatonal.org/">
+                            <img src='@routes.Assets.at("assets/LigaLogo.png")' alt='@Messages("logo.liga.peatonal")'>
+                        </a>
+                    </div>
+                    @if("Columbus" == cityName){
+                        <div class="collaborator-image">
+                            <a id="opencolumbus-img-link" href="https://www.meetup.com/OpenColumbus/">
+                                <img src='@routes.Assets.at("assets/columbus-oh/open_columbus_logo.png")' alt='@Messages("logo.open.columbus")'>
+                            </a>
+                        </div>
+                    }
+                </section>
+                <section class="collaborator-images-container">
                 @if("Amsterdam" == cityName){
                     <div class="collaborator-image">
                         <a id="worldenabled-img-link" href="https://worldenabled.org/">
-                            <img src='@routes.Assets.at("assets/amsterdam/world_enabled.png")' alt='World_Enabled_Logo'>
+                            <img src='@routes.Assets.at("assets/amsterdam/world_enabled.png")' alt='@Messages("logo.world.enabled")'>
                         </a>
                     </div>
                     <div class="collaborator-image">
                         <a id="cityofamsterdam-img-link" href="https://www.amsterdam.nl/en/">
-                            <img src='@routes.Assets.at("assets/amsterdam/city_of_amsterdam.png")' alt='City_Of_Amsterdam_Logo'>
+                            <img src='@routes.Assets.at("assets/amsterdam/city_of_amsterdam.png")' alt='@Messages("logo.city.of.amsterdam")'>
                         </a>
                     </div>
                     <div class="collaborator-image">
                         <a id="amsterdamintelligence-img-link" href="https://www.amsterdamintelligence.com/">
-                            <img src='@routes.Assets.at("assets/amsterdam/amsterdam_intelligence.png")' alt='Amsterdam_Intelligence_Logo'>
+                            <img src='@routes.Assets.at("assets/amsterdam/amsterdam_intelligence.png")' alt='@Messages("logo.amsterdam.intelligence")'>
                         </a>
                     </div>
                 }
-            </section>
+                </section>
+            } else {
+                <section class="collaborator-images-container">
```

Comment:
Can you restructure the if/else statement so that we don't duplicate any HTML? You should be able to do something like what we do for Columbus. Any time we can prevent duplicated code, that's a win!